alarm-clock-soas.yaml

```diff
@@ -1058,6 +1058,32 @@ switch:
     on_turn_off:
       then:
         - script.stop: display_off_script
+  - name: "Display off, on snooze allowed"
+    platform: template
+    id: display_off_on_snooze_allowed
+    optimistic: true
+    restore_mode: RESTORE_DEFAULT_OFF
+    on_turn_on:
```

Comment:
aren't both switches just a setting instead of display off switches? I don't think bot should use the on_turn_on and off triggers